recipes/smart-mode-line-atom-one-dark-theme

```diff
@@ -0,0 +1 @@
+(smart-mode-line-atom-one-dark-theme :repo "daviderestivo/smart-mode-line-atom-one-dark-theme" :fetcher github)
```

Comment:
please add a new-line character at the end of the first line.

Reply:
Thanks ... I will do it.